root/programs/C#/Samples/WebApp_sample/MVC_Sample/MVC_Sample/Views/Crud1/Index.cshtml

```diff
@@ -0,0 +1,127 @@
+﻿@*
+    //**********************************************************************************
+    //* サンプル アプリ・モデル
+    //**********************************************************************************
+
+    // テスト用クラスなので、必要に応じて流用 or 削除して下さい。
+
+    //**********************************************************************************
+    //* クラス名        ：Index
+    //* クラス日本語名  ：CRUDサンプル（Html.BeginForm）
+    //*
+    //* 作成日時        ：－
+    //* 作成者          ：－
+    //* 更新履歴        ：－
+    //*
+    //*  日時        更新者            内容
+    //*  ----------  ----------------  -------------------------------------------------
+    //*  20xx/xx/xx  ＸＸ ＸＸ         ＸＸＸＸ
+    //**********************************************************************************
+*@
+
+@using MVC_Sample.Models.ViewModels
+@model CrudViweModel
+
+@{
+    // ページタイトル
+    ViewBag.Title = "CRUDサンプル（Html.BeginForm）";
+
+    // マスターページ
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@* スクリプト・セクション（RenderSection("HeaderScriptsSection", required: true)に出力されます） *@
+@section HeaderScriptsSection{
+    <!-- Head 部の section -->
+}
+
+@* 本文レイアウト（RenderBodyに出力されます） *@
+@* Html.BeginFormは従来の全体方式、Ajax.BeginFormは部分更新方式（UpdateTargetIdの範囲を部分更新） *@
+@using (Html.BeginForm(null, null, FormMethod.Post, new { id = "form1", onsubmit = "Fx_OnSubmit();" }))
+{
+    @Html.AntiForgeryToken()
+
+    <div class="row">
+        <div class="col-xs-2 col-sm-2 col-md-2 col-lg-2">
```

Comment:
@daisukenishino 

Because a new line begins in the middle of label string, the controls are of unequal position.

![image](https://cloud.githubusercontent.com/assets/9585990/24391163/725a69be-13c8-11e7-8c3f-84c35559604d.png)

Please revise as following: 
```html
<div class="col-xs-4 col-sm-4 col-md-4 col-lg-4">
    ...
</div>
<div class="col-xs-4 col-sm-4 col-md-4 col-lg-4">
    ...
</div>
<div class="col-xs-4 col-sm-4 col-md-4 col-lg-4">
    ...
</div>
```